minio/Caddyfile

```diff
@@ -0,0 +1,8 @@
+your.public.com {
+    proxy / localhost:9000 {
+        proxy_header Host {host}
+        proxy_header X-Real-IP {remote}
+        proxy_header X-Forwarded-Proto {scheme}
+    }
+}
+
```

Comment:
This Caddyfile has just one site in it so it doesn't require nesting. Could be as simple as:

```plain
your.example.com

proxy / localhost:9000 {
    transparent
}
```